cli_reference/odo-important-update.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/odo-important-update.adoc
+
+:_content-type: CONCEPT
+[id="odo-important_update_{context}"]
+include::_attributes/common-attributes.adoc[]
+= Important update on `{odo-title}`
+:context: odo-important-update
+
+toc::[]
+
+Red Hat no longer provides documentation for `{odo-title}`. You can refer the link:https://odo.dev[document] maintained by the upstream community for any information related to `{odo-title}`.
```

Comment:
Saying out loud that Red Hat doesn't provide documentation for one of its projects sounds quite harsh, even if it is technically true. 

I think that this is something that should be reviewed by PM (@mohitsuman).
Maybe we can formulate this some other way.

Reply:
We could also say something along the lines of RH doesn't provide documentation on the OpenShift docs site for odo.

Reply:
Thank you, @pmkovar for your suggestion. 
@mohitsuman Could you please share your input? Thank you!

Reply:
>We could also say something along the lines of RH doesn't provide documentation on the OpenShift docs site for odo.

This sounds much better. 👍

Reply:
Thanks @pmkovar and @kadel for your suggestions. I have updated it as, "Red Hat does not provide information about odo on the OpenShift Container Platform documentation site. You can refer the link:https://odo.dev[document] maintained by the upstream community for any information related to odo."

Let me know your thoughts or approval?